rs/nns/governance/canister/governance.did

```diff
@@ -107,6 +107,18 @@ type ClaimOrRefreshResponse = record {
   refreshed_neuron_id : opt NeuronId;
 };
 
+type RefreshVotingPower = record {
+  // Intentionally left blank.
```

Comment:
Is this left blank because we don't know yet or because there are no arguments? 
I think it would be good if we make a concrete suggestion (as the idea is that we agree on the details of the APIs with the community) or make it explicit if it is supposed to be empty. WDYT?

Reply:
> Is this left blank because we don't know yet or because there are no arguments?

The latter.

Reply:
OK got it, thanks for explaining!